### PR TITLE
Create code.py for Vertical Wordclock guide

### PR DESCRIPTION
**Pull request by TheKitty**

### CODE REVIEW

File: CircuitPython_Simple_Wordclock/code.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+LED13.direction = digitalio.Direction.OUTPUT
+
+pixel_pin = board.D5
+num_pixels = 21
+pixels = neopixel.NeoPixel(pixel_pin, num_pixels, brightness=0.8)
+pixels.fill((0, 0, 0))
+COLOR = (0, 200, 0)  # Green 
+
+# Bitmap values for each value. These can be OR'ed together
+THREE = 0b00000000000000000001
```
Comment (ladyada) [inline CircuitPython_Simple_Wordclock/code.py]: these 0b consts are a little hard to read, please replace with `1 << 0` thru `1 << 15` (or whatever)

Comment (TheKitty) [reply CircuitPython_Simple_Wordclock/code.py]: They were very easy to see before Mu didn't like the spaces so they lined up showing the bit pattern.

Comment (ladyada) [reply CircuitPython_Simple_Wordclock/code.py]: and hard for people to change around the bits - we dont need to save any space here :)

Comment (TheKitty) [reply CircuitPython_Simple_Wordclock/code.py]: roger, changed

**Pull request merged**